solr/core/src/java/org/apache/solr/handler/admin/ContainerPluginsApi.java

```diff
@@ -51,7 +51,7 @@
 public class ContainerPluginsApi {
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
-  public static final String PLUGIN = "plugin";
+  public static final String PLUGINS = "plugin";
```

Comment:
why change the variable name at all?

Reply:
it's a left-over from the earlier change that reverted but I missed this change.

Still, I think we should change the name of the path to "plugins" because the singular form doesn't make sense. Back-compat can be preserved by initially accepting both names.

Reply:
If you plan to change it, it should be a part of a different PR. We can remove that change from here

Reply:
I mean why these changes should be present in this PR?

Reply:
Removed.